### PR TITLE
[SC-243] Rename cfn-cr-synpase-tagger to cfn-cr-sc-tagger

### PR DESCRIPTION
**Pull request by cascadianblue**

### CODE REVIEW

**Review by brucehoff — APPROVED** [review]

**Review by zaro0508 — CHANGES_REQUESTED** [review]
Please do not merge yet.  This is out of date, we'll need to bring it up to date and I would like to see all dependent PRs for this before merging.

**Comment by zaro0508** [comment]
I've decided this is too much work for little value.  Won't do.

**Pull request closed without merging**
